### ISSUE TITLE
highlight.js emits warning if no language is set on source block

### ISSUE THREAD

**Issue by mojavelinux** [asciidoctor/asciidoctor#4263]
If no language is specified on a source block, and the `source-highlighter` attribute is set to `highlight.js`, the highlight.js emits the following warning into the browser console:

```
Could not find the language 'none', did you forget to load/include a language module?
Falling back to no-highlight mode for this block.
```

Here's an example:

```
[source]
----
no lang
----
```

The warning happens because the language `none` is not mapped to `plaintext`. To fix it, either add the mapping for `none` or don't pass blocks with the language `none` to highlight.js.

**Comment by mojavelinux** [comment]
Backport the fix for this issue to the v2.0.x branch.

**Comment by mojavelinux** [comment]
Another solution is to add the CSS class `language-plaintext` so that highlight.js resolves the language to `plaintext`, which is mapped.

```html
<div class="listingblock">
<div class="content">
<pre class="highlightjs highlight"><code class="language-plaintext language-none hljs">no lang</code></pre>
</div>
</div>
</div>
```

**Comment by mojavelinux** [comment]
Resolved by d5c9ab50a4ced8a1ee8499cb453d67f8d1b3fe6a and backported.
